themes/gatsby-theme-wordpress-blog/gatsby-config.js

```diff
@@ -11,8 +11,8 @@ module.exports = (options) => ({
       options,
     },
     "gatsby-transformer-remark",
-    "gatsby-plugin-image",
-    "gatsby-plugin-sharp",
     "gatsby-transformer-sharp",
+    "gatsby-plugin-sharp",
+    "gatsby-plugin-image",
```

Comment:
Just clarifying for my own understanding — order does not matter here, correct?

Reply:
Shouldn't matter. I moved this to double check that this wasn't causing anything related to the WpUser's gravatar to not work, but it's all working as expected